src/otx/v2/adapters/torch/mmengine/mmdet/modules/models/heads/custom_roi_head.py

```diff
@@ -0,0 +1,313 @@
+"""Custom ROI head for OTX template."""
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import torch
+from mmdet.models.losses import accuracy
+from mmdet.models.roi_heads.bbox_heads.convfc_bbox_head import Shared2FCBBoxHead
+from mmdet.models.roi_heads.standard_roi_head import StandardRoIHead
+from mmdet.models.utils import multi_apply, unpack_gt_instances
+from mmdet.registry import MODELS
+from mmdet.structures.bbox import bbox2roi
+from torch import Tensor
+
+if TYPE_CHECKING:
+    from mmdet.models.task_modules.samplers import SamplingResult
+    from mmdet.structures import DetDataSample
+    from mmdet.utils import InstanceList
+    from mmengine.config import Config, ConfigDict
+
+
+from otx.v2.adapters.torch.mmengine.mmdet.modules.losses.cross_focal_loss import (
+    CrossSigmoidFocalLoss,
+)
+from otx.v2.adapters.torch.mmengine.mmdet.modules.models.heads.cross_dataset_detector_head import (
+    CrossDatasetDetectorHead,
+)
+
+
+@MODELS.register_module()
+class CustomRoIHead(StandardRoIHead):
+    """CustomROIHead class for OTX."""
```

Comment:
Is it possible to add a brief description, why we need a custom RoI Head, and what exactly we want to achive with this implementation.